docs/measurements/index.md

```diff
@@ -47,18 +47,28 @@ model = QuantumModel(
 
 # Define a measurement protocol by passing the shot budget as an option.
 tomo_options = {"n_shots": 100000}
-tomo_measurement = Measurements(protocol=Measurements.TOMOGRAPHY, options=tomo_options)
+tomo_measurement = Measurements(protocol=MeasurementProtocols.TOMOGRAPHY, options=tomo_options)
 
 # Get the exact expectation value.
 exact_values = model.expectation()
 
 # Run the tomography experiment.
-estimated_values_tomo = tomo_measurement(model)
+estimated_values_tomo = tomo_measurement(model, return_expectations=True)
```

Comment:
Is there a need to pass on this argument ?

Reply:
No, just showing it for clarity